### ISSUE TITLE
Question: Does this allow entirely local communcation?

### ISSUE THREAD

**Issue by jkms** [garbled1/homeassistant_ecowitt#85]
I want to buy a weatherstation that I can connect to my IOT newtork which does not have cloud access. Is this possible with this addon?

**Comment by garbled1** [comment]
Yes,  there is no cloud needed at all with this integration.  (Which is why I use it)

**Comment by Anto79-ops** [comment]
what format is the data being sent out as? MQTT? say if I want the sensor data to go out to influxDB (and Grafana), what do I need to add to make that connection NodeRed?

**Comment by adorobis** [comment]
> what do I need to add to make that connection NodeRed?

Just configure [influxdb integration](https://www.home-assistant.io/integrations/influxdb/) in HA and all data that you wish will be sent to it.

**Comment by fsevilla3** [comment]
I have an [Ecowitt GW1000 Wireless Weather Station Gateway](https://www.ecowitt.com/shop/goodsDetail/107) integrated with a bunch of PM2.5, soil moisture, and temperature and humidity sensors. I use the [Ecowitt Weather Station HACS integration](https://github.com/garbled1/homeassistant_ecowitt) for local integration, and the GW1000 is configured to send readings to my local Home Assistant server using its private IP address (not hostname) on the custom port that the HACS integration listens on. I had isolated the GW1000 from the Internet by moving it to a local-only segment of my network that can speak with other local devices but not the Internet.

This setup worked fine for almost a year, until I misguidedly updated the GW1000 firmware to v1.6.9 after having seen the update available on the app. Now, the GW1000 only works if it has Internet access. It works for a short period of time after being isolated from the Internet (approximately 15 - 20 minutes) after which the sensors appear offline both in the WS View Plus app and Home Assistant. The issue is resolved within a couple of minutes after re-enabling Internet access for the GW1000.

This isn't an issue with the HACS integration, but I thought I'd share it here as I think it's relevant to your question. If anyone stumbles across this and has fixed this issue, please let me know. Thanks!
